client/src/components/layout/TopBar.js

```diff
@@ -15,6 +15,9 @@ const TopBar = ({ user }) => {
   ];
 
   const authenticatedListItems = [
+    <li className="username">
```

Comment:
this is generating a warning in the browser console that should be resolved - each element in a list should have a unique `key` prop